src/js/engagement_view/src/components/graphDisplay/GraphDisplay.tsx

```diff
@@ -218,7 +210,6 @@ const GraphDisplay = ({ lensName, setCurNode }: GraphDisplayProps) => {
                 98,
                 NODE_R / ctx.measureText(label).width
             );
```

Comment:
and just to make sure, all this removal of ctx is expected?

Reply:
Yes